stubs/click/click/core.pyi

```diff
@@ -151,7 +153,6 @@ _T = TypeVar("_T")
 _F = TypeVar("_F", bound=Callable[..., Any])
 
 class MultiCommand(Command):
-    no_args_is_help: bool
```

Comment:
I guess this should also be deleted from `MultiCommand`'s `__init__`?

Reply:
Actually, we shouldn't delete this from `MultiCommand`, it should be in both classes like [it is here](https://github.com/pallets/click/blob/master/src/click/core.py)

Reply:
Oh, I didn't see this...

Wait a minute.

Reply:
> Actually, we shouldn't delete this from `MultiCommand`, it should be in both classes like [it is here](https://github.com/pallets/click/blob/master/src/click/core.py?rgh-link-date=2021-04-10T14%3A08%3A20Z)

Yes, you are right, we should keep this argument in MultiCommand constructor.